src/homework/homework02/Student.java

```diff
@@ -1,24 +1,51 @@
 package homework.homework02;
 
-public class Student extends Person{
+public class Student {
+    private Faculty faculty;
     private int semester;
-    public int getSemester() {
-        return semester;
-    }
-    public void setSemester(int semester) {
+    private Person person;
+
+    public Student(Faculty faculty, int semester, Person person) {
+        this.faculty = faculty;
         this.semester = semester;
+        this.person = person;
     }
 
-    private Faculty faculty;
+    public String getName() {
+        return person.name;
+    }
+
+    public void setName(String name) {
+        person.setName(name);
+    }
+
+    public String getSurname() {
+        return person.surname;
+    }
 
-    public Student (int pesel, String name, String surname, Faculty faculty, int semester) {
-        super (pesel, name, surname);
+    public void setSurname(String surname) {
+        person.setSurname(surname);
+    }
+
+    public int getPesel() {
+        return person.pesel;
+    }
+
+    public void setPesel(int pesel) {
+        person.setPesel(pesel);
+    }
+
+    public int getSemester() {
+        return semester;
+    }
+
+    public void setSemester(int semester) {
         this.semester = semester;
     }
 
-    public void studentInfo() {
-        super.personInfo();
-        System.out.println("faculty: " + faculty + "\nsemester: " + semester);
+    public String getFaculty() {
```

Comment:
Nie jest jasne: czy chcesz zwrócić faculty (bo na to wskazuje nazwa), czy tego facultetu nazwę (bo na to wskazuje ciało)

Reply:
Skąd Ty to wygrzebałeś? Wyrzuciłam wszystkie ...Info (albo zmieniłam na getInfo bez ciała i w komentarzu, żeby pamiętać, że miałam dokończyć)

Reply:
Muhihi.

GitHub był nadgorliwy :)

Ja komentowałem tylko getFaculty, a ten się dopatrzył, że w tym miejscu kiedyś było studentInfo (a usunięcie jeszcze nie zostało zmergowane), to napisał, że było. Nawet nie miałem wiedzy, że on to zacytuje :)

Reply:
O, faktycznie jest na czerwono ;)
Podmieniłam nazwę metody.